### DISCUSSION TITLE
Optics Council Call

### DISCUSSION THREAD

**Discussion by YazzyYaz** [celo-org/celo-proposals#359] [CIP Discussions]
### Optics Council Call

![0_LzqzC_gbzgureeL5](https://user-images.githubusercontent.com/9094204/145140760-969099a6-7db8-4b74-bb47-6f8f618d4faa.jpeg)

The Optics Council Call will be held to discuss short and medium-term development direction for Optics with Celo dApps.

Time: Dec 9, 2021, 01:00 PM Eastern Time (US and Canada)
Join Zoom Meeting: https://us06web.zoom.us/j/87690048999
Meeting ID: 876 9004 8999
Add To Calendar [here](https://calendar.google.com/event?action=TEMPLATE&tmeid=MXM5OGllajR0Z20wYmJlOW1kN2htNzMzcjMgY185c3U2aWNoMXVobWV0cjRvYjNzaWo2a2Fxc0Bn&tmsrc=c_9su6ich1uhmetr4ob3sij6kaqs%40group.calendar.google.com)

**Comment by aaronmgdr** [comment]
1. Short-term reliability
* Tx submission (EIP1559)
* operational monitoring
2. Security
* Watchers
* Optics recovery mode indication on UI
* Monitoring upgrades or recovery mode initiation bots
3. Expansion
* new Chains

(from @nambrot )

**Comment by aaronmgdr** [comment]
notes from call. 

1. Status Update. 

* optics v2  token bridge launched last week
* v2 TVL about 10m 
* patrick: suggest defillama listing 
* clabs running agents for that deployment
* mobius and clabs have launced v2 ui
* supporting migrations
* foundation liquidity for v2 

nam:
* made improvements to reliability of agents

* deployed Avalanch with v2 but need to make live (its hidden in ui but contacts are there)

# Ops reliability
* gas spikes in eth require manual work to keep it going 
* need more monitoring 
* watchers! 
*

# Incentives? 
* no insentives righ now
* moving to full model that has incentives. lock funds, slashing, POS,  
* security improvments from single key for correct head of mercle root to a 2 / 3 or 3/5 
* watchers are the weakest point atm





# QA 
* who works on optics 
* value in optics being run by entity apart from clabs. 
*  7-8 clabs people working full to part time on optics (up to 12) 
* cLabs commited to optics LT success. 
* at same time clabs does not want to be only org working on optics. 

# whats timeline and roadmap for decentralization?
* hiring more people, 
* goal of dedicated team at clabs and amoung other organizations 
 
# can we push watcher out to other existing validators? dapps, dexs? 
* serious process for updating contracts. 

..droping off for another meeting
